Makefile

```diff
@@ -17,20 +16,27 @@ capture.so: capture.c
 	sed 's/@PREFIX@/$(subst /,\/,$(PREFIX))/' $+ > $@
 
 install: all
-	install -d $(DESTDIR)$(PREFIX)/bin
-	install -d $(DESTDIR)$(PREFIX)/share/adi_colorimeter/
-	install -d $(DESTDIR)$(PREFIX)/lib/adi_colorimeter/
-	install ./org.adi.pkexec.adi_colorimeter.policy /usr/share/polkit-1/actions 
-	install ./adi_colorimeter $(DESTDIR)$(PREFIX)/bin/
-	install ./capture.so $(DESTDIR)$(PREFIX)/lib/adi_colorimeter/
-	install ./adi_colorimeter.glade $(DESTDIR)$(PREFIX)/share/adi_colorimeter/
-	./setup.py install --prefix=$(PREFIX) --root=$(DESTDIR)
+	install -d $(PREFIX)/bin
+	install -d $(PREFIX)/share/adi_colorimeter/
+	install -d $(PREFIX)/lib/adi_colorimeter/
+	install -d $(PREFIX)/share/adi_colorimeter/icons/
+	install ./org.adi.pkexec.adi_colorimeter.policy /usr/share/polkit-1/actions/
+	install ./adi_colorimeter $(PREFIX)/bin/
+	install ./capture.so $(PREFIX)/lib/adi_colorimeter/
+	install ./adi_colorimeter.glade $(PREFIX)/share/adi_colorimeter/
+	install ./icons/ADIlogo.png $(PREFIX)/share/adi_colorimeter/icons/
```

Comment:
Actually, lines:
`install -d $(PREFIX)/share/adi_colorimeter/icons/`
and 
`install ./icons/ADIlogo.png $(PREFIX)/share/adi_colorimeter/icons/`
should be part of the commit that adds the PNG file.